### DISCUSSION TITLE
Increase bar cursor thickness a bit to be more visible

### DISCUSSION THREAD

**Discussion by rsj-ioki** [helix-editor/helix#7340] [Q&A] [answered]
I use following cursor related config like many helix users:
```
[editor.cursor-shape]
insert = "bar"
normal = "block"
select = "underline"
```
So while in insert mode and especially on lines with empty spaces it is a bit hard to see the cursor bar as it is too thin. Following is what I see using `gruvbox` and `gruvbox_light` themes:

![Screenshot from 2023-06-13 23-00-57](https://github.com/helix-editor/helix/assets/68541057/5f579fba-4922-4b80-b4db-69a01001b85a)

![Screenshot from 2023-06-13 23-01-58](https://github.com/helix-editor/helix/assets/68541057/a3bb38b9-4df7-4557-9ddd-ccb448d14e9f)

It looks to me like increase a thickness a bit can improve the situation. For now I have `cursorcolumn = true` in my config to help me.

Version: `helix 23.05 (7f5940be)`

**Comment by the-mikedavis** [comment, marked as answer]
The rendering of the bar cursor is controlled by your terminal. For example you can configure the thickness in kitty in the kitty config: https://sw.kovidgoyal.net/kitty/conf/#cursor-customization

**Reply by rsj-ioki** [reply]
I guess that is not possible with gnome terminal :cry: but thank you for pointing this out.
